app/models/building.rb

```diff
@@ -0,0 +1,3 @@
+class Building < ApplicationRecord
+  include Timeable
+end
```

Comment:
Why is this empty? How does it work with no fields specified? o.O We are confused :D
(same for timeable.rb and rooms.rb)